tests/chainer_tests/test_function.py

```diff
@@ -343,11 +348,25 @@ def test_debug_forward_gpu(self):
             return_value=(cuda.to_gpu(self.nan),))
         self.check_debug_forward(cuda.to_gpu(self.one))
 
-    def check_debug_backward(self, x_data):
+    def test_debug_forward_int_cpu(self):
+        self.f.forward_cpu = mock.MagicMock(return_value=(self.int_one,))
+        self.check_debug_forward(self.one, valid=True)
```

Comment:
Can we make `test_debug_forward_cpu` and `test_debug_forward_int_cpu` parameterized with respect to `return_value` and `valid`?